pkg/report/report_test.go

```diff
@@ -70,11 +70,23 @@ func testParseFile(t *testing.T, reporter Reporter, fn string) {
 				test.HasReport = true
 				phase = phaseReport
 			} else {
-				test.Log = append(test.Log, s.Bytes()...)
+				// Strip all \r from reports because the merger removes it.
```

Comment:
If it will be less duplication and will handle all current and future cases if in the beginning of the function you replace:
```
	input, err := os.Open(fn)
	s := bufio.NewScanner(input)
```
with:
```
	data, err := ioutil.ReadFile(fn)
	// Because ...
	data = bytes.ReplaceAll(data, []byte{'\r'}, nil)
	s := bufio.NewScanner(bytes.NewReader(input))
```

Reply:
Without bytes.IndexByte. That is only an optimization, for the test we don't care.

Reply:
fixed.